some startup errors

after installing koa-locales, I got the error message "Cannot find module 'merge-descriptors'".
Then I manually moved "merge-descriptors" from devDependencies to dependencies, and after hitting npm install in that folder "node_modules/koa-locales/" I get this:

```
.../node_modules/koa-locales/index.js:98
  app.context[functionName] = function (key, value) {

TypeError: Cannot set property '__' of undefined
```
I am stuck.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,4 @@
+sudo: false
 language: node_js
 node_js:
   - '4'

--- a/package.json
+++ b/package.json
@@ -17,21 +17,21 @@
   "dependencies": {
     "debug": "~2.2.0",
     "humanize-ms": "~1.0.1",
-    "ini": "~1.3.4"
+    "ini": "~1.3.4",
+    "merge-descriptors": "~1.0.0"
   },
   "devDependencies": {
     "autod": "*",
-    "beautify-benchmark": "~0.2.4",
-    "benchmark": "~1.0.0",
+    "beautify-benchmark": "0",
+    "benchmark": "1",
     "contributors": "*",
     "istanbul-harmony": "*",
     "jshint": "*",
-    "koa": "~0.21.0",
-    "mm": "~1.1.0",
+    "koa": "1",
+    "mm": "1",
     "mocha": "*",
-    "pedding": "~1.0.0",
-    "supertest": "~1.0.1",
-    "merge-descriptors": "~1.0.0"
+    "pedding": "1",
+    "supertest": "1"
   },
   "homepage": "https://github.com/koajs/locales",
   "repository": {

